juniper_official/Solutions/EVPN-VXLAN/check-bpdu-errors-netconf.rule

```diff
@@ -0,0 +1,93 @@
+/*
+ * Collects lacp statistics periodically and notifies anomalies when lacp  
+ * distribution status is false.
+ *
+ * One input control detection
+ * 
+ *  1) ae-name, is the interface name to monitor. By default monitors all AE
+ *     interfaces. For specific interfaces to monitor, use regular expression
+ *     for e.g. ae1 or ae1|ae1.
+ */
```

Comment:
s/One input control detection/One input controls detection/g
s/ae-name,/ae-name variable,/g

Reply:
updated the rule